vsts/pipelines/templates/_buildParallel.yml

```diff
@@ -109,6 +109,11 @@ steps:
   inputs:
     scriptPath: ./build/test-integration.sh
     args: $(TestIntegrationCaseFilter)
+  env:
```

Comment:
This works, but this is not ideal as we do not want to expose the secrets as plain text in environment variables (it defeats the purpose of holding secrets in keyvault). Seems like the recommended way to access keyvault is through Manage Service Identity which does not work for our scenario of using them in tests. Are we OK with this risk vs. the earlier idea that I had about running a dedicated sql server container in the build machine? cc gpcastro